src/toast.toasts.ts

```diff
@@ -0,0 +1,352 @@
+import './icon-button.js';
+import { html, LitElement } from 'lit';
+import { createRef, ref } from 'lit/directives/ref.js';
+import { customElement, property } from 'lit/decorators.js';
+import { repeat } from 'lit/directives/repeat.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { choose } from 'lit/directives/choose.js';
+import { when } from 'lit/directives/when.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import xIcon from './icons/x.js';
+import { LocalizeController } from './library/localize.js';
+import styles from './toast.toasts.styles.js';
+import shadowRootMode from './library/shadow-root-mode.js';
+import final from './library/final.js';
+import GlideCoreToast from './toast.js';
+import GlideCoreLink from './link.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'glide-core-private-toasts': GlideCoreToasts;
+  }
+}
+
+/**
+ * @attr {GlideCoreToast[]} [toasts=[]]
+ *
+ * @method showPopover
+ */
+@customElement('glide-core-private-toasts')
```

Comment:
Still two components: Toast and Toasts. The difference in how it works not is Toast is what the consumer uses and Toasts is what we use, and it's private.